pkg/defaults.py

```diff
@@ -0,0 +1,18 @@
+#-------------------------------------------------------------------------------
+#    Dictionary of fpm keyword arguments that are common to all packages
+#-------------------------------------------------------------------------------
+
+OPTS.update(
+    name = "dfmt",
+    url = 'https://github.com/dlamg-community/dfmt',
+    maintainer = 'Stefan Koch <stefan.koch@sociomantic.com>',
+    vendor = 'Sociomantic Labs GmbH',
+    provides = "dfmt",
+)
+
+#-------------------------------------------------------------------------------
+#    List of fpm positional arguments that are common to all packages
+#-------------------------------------------------------------------------------
+
+#ARGS = [ ]
+
```

Comment:
This separation is not useful if you only have one package. I would do everything in `dfmt.pkg`.